tests/gp_tests/test_search_space.py

```diff
@@ -92,31 +89,25 @@ def test_round_one_normalized_param(
 
 def test_sample_normalized_params() -> None:
     search_space = SearchSpace(
-        scale_types=np.array(
-            [
-                ScaleType.LINEAR,
-                ScaleType.LINEAR,
-                ScaleType.LOG,
-                ScaleType.LOG,
-                ScaleType.CATEGORICAL,
-            ]
-        ),
-        bounds=np.array([(0.0, 10.0), (1.0, 10.0), (10.0, 100.0), (10.0, 100.0), (0.0, 5.0)]),
-        steps=np.array([0.0, 1.0, 0.0, 1.0, 1.0]),
-    )
-    samples = sample_normalized_params(
-        n=128, search_space=search_space, rng=np.random.RandomState(0)
+        {
+            "a": optuna.distributions.FloatDistribution(0.0, 10.0, False, None),
+            "b": optuna.distributions.IntDistribution(1, 10),
+            "c": optuna.distributions.FloatDistribution(10.0, 100.0, True, None),
+            "d": optuna.distributions.IntDistribution(10, 100, True),
```

Comment:
[nit] Better to specify the keywords.
```suggestion
            "a": optuna.distributions.FloatDistribution(0.0, 10.0),
            "b": optuna.distributions.IntDistribution(1, 10),
            "c": optuna.distributions.FloatDistribution(10.0, 100.0, log=True),
            "d": optuna.distributions.IntDistribution(10, 100, log=True),
```